src/app/Components/StickyTabPage/StickyTabPageFlatList.tsx

```diff
@@ -93,11 +96,19 @@ export const StickyTabPageFlatList: React.FC<StickyTabFlatListProps> = (props) =
   // to avoid jumping content in situations where certain items have fixed height.
   // This doesn't make a lot of sense but sometimes you just go with what works ¯\_(ツ)_/¯
   const [headerDidMount, setHeaderDidMount] = useState(__TEST__)
-  const { data, style, ...otherProps } = props
+  const { data, style, refresh, isRefreshing, ...otherProps } = props
 
   return (
     <Animated.View style={{ flex: 1, paddingTop: totalStickyHeaderHeight }}>
       <AnimatedFlatList
+        refreshControl={
+          <RefreshControl
+            progressViewOffset={refreshControllPadding ?? 0}
+            refreshing={!!isRefreshing}
+            onRefresh={refresh || null || undefined}
```

Comment:
```suggestion
            onRefresh={refresh}
```
Because onRefresh should be `() => void | undefined`

Reply:
You're right, thanks!